src/test/java/uk/nhs/tis/sync/job/reval/RevalCurrentPmSyncJobIntegrationTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.nhs.tis.sync.job.reval;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.verify;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.awaitility.core.ConditionTimeoutException;
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.Sql.ExecutionPhase;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.nhs.tis.sync.message.publisher.RabbitMqTcsPmUpdatePublisher;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+//?Need this?(classes = Application.class, webEnvironment = SpringBootTest.WebEnvironment.MOCK)
```

Comment:
Please address this commented code

Reply:
Don't think we need them, so will remove this line.